vocode/streaming/models/synthesizer.py

```diff
@@ -45,6 +45,7 @@ class SynthesizerConfig(TypedModel, type=SynthesizerType.BASE.value):
     audio_encoding: AudioEncoding
     should_encode_as_wav: bool = False
     sentiment_config: Optional[SentimentConfig] = None
+    text_to_speech_chunk_size_seconds: float = 1.0
```

Comment:
If we know we want a default of <1 here I think we can just include it in this PR?

Reply:
offline discussed: It is on other PR